src/language-js/utils.js

```diff
@@ -11,6 +11,20 @@ const {
 } = require("../common/util");
 const handleComments = require("./comments");
 
+/**
+ * @typedef {import("estree").BaseNode} BaseNode
+ * @typedef {import("estree").BaseExpression} BaseExpression
+ * @typedef {import("estree").Expression} Expression
+ * @typedef {import("estree").MemberExpression} MemberExpression
+ *
+ * @typedef {Object} MemberExpressionNode
+ * @mixes MemberExpression
+ *
+ * @typedef {Object} TemplateLiteralNode
+ * @mixes BaseNode
+ * @property {MemberExpressionNode[]} expressions
```

Comment:
TemplateLiteral.expression could be  any  estree  node, I  don't think `MemberExpressionNode[]` is  right.

Reply:
https://github.com/estree/estree/blob/master/es2015.md#templateliteral